synapse/handlers/devicemessage.py

```diff
@@ -221,17 +227,15 @@ async def send_device_message(
 
         remote_edu_contents = {}
         for destination, messages in remote_messages.items():
-            with start_active_span("to_device_for_user"):
-                set_tag("destination", destination)
-                remote_edu_contents[destination] = {
-                    "messages": messages,
-                    "sender": sender_user_id,
-                    "type": message_type,
-                    "message_id": message_id,
-                    "org.matrix.opentracing_context": json_encoder.encode(context),
-                }
+            log_kv({"destination": destination})
+            remote_edu_contents[destination] = {
+                "messages": messages,
+                "sender": sender_user_id,
+                "type": message_type,
+                "message_id": message_id,
+                "org.matrix.opentracing_context": json_encoder.encode(context),
+            }
 
-        log_kv({"local_messages": local_messages})
```

Comment:
Looks like the new `log_kv({user_id, device_id})` above is meant to be the replacement for this `log_kv` and the one below. Is that correct?

Reply:
Yup. This log was a bit useless as it gave the entire message, which then got truncated by jaeger, so you couldn't really see what was going on